protocol/types.yml

```diff
@@ -207,6 +207,14 @@ DirectoryQueryResult:
     type: slice
     slice:
       type: string
+    optional: true
+  - name: ExpandedEntries
+    type: slice
+    slice:
+      type: state.Object
+      pointer: true
+      marshal-as: self
```

Comment:
I had been assuming the type generator would trip on references to types from other packages. I'm happy to know that it works.